### DISCUSSION TITLE
ImportError: cannot import name 'ISO_8601' from 'rest_framework' (unknown location)

### DISCUSSION THREAD

**Discussion by qcymkxyc** [encode/django-rest-framework#8069] [Question & Answer]
I tried many times,when I run app this error appear

**Comment by lovelydinosaur** [comment]
You haven't described clearly enough what you're doing for anyone to be able to help you.

**Reply by mrhaoll** [reply]
when i install the old rest-framework package, i also get the problem.  then i remove all the rest framework package, and reinstall the django-rest-framework, the program work normal now
